Apps/Database/src/Delegates/DBProfileDelegate.cs

```diff
@@ -161,7 +161,7 @@ public int GeUnregisteredInvitedUsersCount()
         public int GetLoggedInUsersCount()
         {
             int result = this.dbContext.UserProfile
-                .Count(u => u.LastLoginDateTime >= DateTime.Today.ToUniversalTime());
+                .Count(u => u.LastLoginDateTime.HasValue && u.LastLoginDateTime.Value >= DateTime.UtcNow.Date);
```

Comment:
Why greater or equal? Also, the ufc offset could count users from the previous night as logged in today and the ones in the afternoon as counted for tomorrow.

Reply:
If the last login is recorded as utc there shouldnt be an issue, query should be:

Reply:
LastLoginDateTime (2020-03-27 13:00) >= Current Date (2020-03-27 00:00) both converted to utc

Reply:
On UTC the comparison is correct. However the admin that needs to see if it is on the same day on its own time zone.